lib/api/v1/user-controller.js

```diff
@@ -34,15 +35,19 @@ module.exports = function(app, user) {
   app.use(express.json());
 
   // Get a list of all Users
-  app.get('/api/v1/user', user.can('manage users'), function(req, res) {
-    User.find({}, '-password', function(err, users) {
+  app.get('/api/v1/user', user.can('view users'), function(req, res) {
+    var query = {};
+    if (req.user && (new UserPermissions(req.user)).can('not view other users'))
```

Comment:
I've never seen permissions defined in the negative like this. I think it's probably not a good idea. I assume this is done to handle the viewer role being able to see their own profile. But why would they ever need to list _all_ users? Could you just mark this one 'view users' and be done with it? Or if it's really needed, then I'd suggest 'view other users' and 'view profile' permissions.